File: backend/services/configuration-service/src/main/java/com/lmspilot/configuration/ConfigurationServiceApplication.java
package com.lmspilot.configuration;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;


@SpringBootApplication(scanBasePackages = "com.lmspilot")
public class ConfigurationServiceApplication {
    public static void main(String[] args) { SpringApplication.run(ConfigurationServiceApplication.class, args); }
}

File: backend/services/configuration-service/src/main/kotlin/com/lmspilot/configuration/ConfigurationServiceApplication.kt
package com.lmspilot.configuration

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.lmspilot"])
class ConfigurationServiceApplication

fun main(args: Array<String>) {
    runApplication<ConfigurationServiceApplication>(*args)
}
